src/main/java/org/jabref/migrations/PreferencesMigrations.java

```diff
@@ -301,6 +302,12 @@ static void upgradePreviewStyleFromReviewToComment(JabRefPreferences prefs) {
         prefs.setPreviewStyle(migratedStyle);
     }
 
+    static void upgradePreviewStyleAllowMarkdown(JabRefPreferences prefs) {
+        String currentPreviewStyle = prefs.getPreviewStyle();
+        String migratedStyle = currentPreviewStyle.replace("\\begin{comment}<BR><BR><b>Comment: </b> \\format[HTMLChars]{\\comment} \\end{comment}", "\\begin{comment}<BR><BR><b>Comment: </b> \\format[Markdown,HTMLChars]{\\comment} \\end{comment}");
```

Comment:
Is there a reason not to simply replace `\\format[HTMLChars]{\\comment}` with `\\format[Markdown,HTMLChars]{\\comment}` (or `\\format[Markdown]{\\comment}` see above)? If the user changed the preview style a bit, then this replacement would still have the desired effect, right?

Reply:
My reasoning was to not force a different behavior on people who consciously changed their configuration. So I only changed it, when it's the unchanged default value. However this is a matter of opinion. How are changes like this normally handled in Jabref?

Reply:
I like the proposal by @tobiasdiez. It a) integrated the feature and b) focuses the update to the `Comment` field